packages/flutter/lib/src/gestures/pointer_router.dart

```diff
@@ -71,6 +71,18 @@ class PointerRouter {
     _globalRoutes.remove(route);
   }
 
+  /// The number of global routes that have been registered.
+  ///
+  /// This will be non-null (and valid) in debug builds only.
+  int? get debugGlobalRouteCount {
```

Comment:
if it's always non-null in debug builds and always invalid in release builds make it non-nullable and throw in non-debug builds

Reply:
Done